src/containers/List/ListFilters/ListFilters.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect, useState } from 'react'
+import { Dropdown } from 'semantic-ui-react'
+import { useRouter } from '../../../utils/hooks/useRouter'
+import { FilterDefinitions } from '../../../utils/types'
+import BooleanFilter from './BooleanFilter'
+import { EnumFilter, SearchableListFilter, StaticListFilter } from './OptionFilters'
+import { GetMethodsForOptionFilter } from './types'
+
```

Comment:
Just putting this here, not sure where it belongs:

Seems like a bug on the filter list drop-down: when first loading, the drop-down has an additional "check" item at the top of the list, which does nothing. It disappears after selecting it.
![Screen Shot 2021-06-22 at 11 05 42 AM](https://user-images.githubusercontent.com/5456533/122838373-0f242600-d34a-11eb-937c-2e397998a711.png)

Reply:
e572da065eab52ada930e0b46201740ee4af7115